views/components/organisms/o-header.hbs

```diff
@@ -0,0 +1,39 @@
+<header>
+		<nav class="row nav">
+				<div class="col span-1-of-4">
+				{{> atoms/a-img class="nav__logo" imageUrl="/static/images/logo.png" alt="Sab ka Bazar" onclick="movetoHome('/')"}}
+				</div>
+				<div class="col span-2-of-4">
+					<ul class="home-nav">
```

Comment:
As we dont have multi level navigation, we can avoid use of ul and li. it will still be a valid markup and we will save on the DOM size too

Reply:
I have removed ul and li. Thanks.